docs/instrument.md

```diff
@@ -0,0 +1,49 @@
+
+## Producing instrumented source
+
+The `nyc instrument` command can produce instrumented source files.
+These files are suitable for client side deployment during end to end testing.
+You can either pre-instrument your source, or write instrumented source to a stream.
+
+Run `nyc instrument --help` to display a full list of available command options.
+ 
+### Pre-instrumented source 
+ 
+You can create pre-instrumented source code by running:
+
+```bash
+nyc instrument <input> [output]
+```
+
+`<input>` can be any file or directory within the project root directory.
+The `[output]` directory is optional and can be located anywhere, if not set the instrumented code will be sent to `stdout`.
```

Comment:
maybe specify that these should be relative paths?

Reply:
After running a few tests, it works with both relative and absolute paths, on OSX at least.

Reply:
True... Curious how it would behave on windows